addons/arsenal/functions/fnc_sortPanel.sqf

```diff
@@ -17,57 +17,147 @@
 params ["_control", "_mode"];
 
 private _display = ctrlParent _control;
-private ["_panel", "_curSel", "_selected"];
 
-// Right panel
-if (ctrlIDC _control == 17 && {GVAR(currentLeftPanel) in [IDC_buttonUniform ,IDC_buttonVest, IDC_buttonBackpack]}) then {
-    _panel = _display displayCtrl IDC_rightTabContentListnBox;
-    _curSel = lnbCurSelRow _panel;
-    _selected = _panel lnbData [_curSel, 0];
-
-    switch (_mode) do {
-        case 0: {
-            _panel lnbSort [1, false];
+private _to_string = {
```

Comment:
Can use `CBA_fnc_formatNumber` for this?

Reply:
Wasn't array of that function, looks like a better solution